controller/job.go

```diff
@@ -54,3 +54,24 @@ func (jc *JobController) GetJobsFromAPI(
 	json.NewEncoder(w).Encode(response)
 
 }
+
+// GetJobsConcurrently function
+func (jbc *JobController) GetJobsConcurrently(w http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
func (jc *JobController) GetJobsConcurrently(w http.ResponseWriter, r *http.Request) {
```
Be consistent in your receiver's variables.